syntax = "proto2";

message TopicConfig {
    optional string camera_merged_topic = 3;
    optional string pose_estimation_result_topic = 4;
}

message Config {
    optional TopicConfig topic_config = 1;
}
